### PR TITLE
docs(README.md): update readme package docs

### PR DESCRIPTION
**Pull request by bad4iz**
## Description

update readme package docs

## How Has This Been Tested?

none 

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/docs/README.md [hunk 1]
```diff
@@ -1,60 +1,181 @@
...
+In git, add all relevent files.
 
-If you can not completely finish a translation, but want to send the partial work, make it clear in the _issue_ of your work after the submission.
+Commit with commitizen using the command `git cz`. From here you will follow through a series of props. Make sure to select the appropiate type (see **Commit Guidelines w/Commitizen** above)
+
+Lastly, `git push` and open a pull request.
+
+### Pull Requests For Vuetify
+
+> Pull requests related to Vuetify: for bug fixes submit pull requests to `master`, for new features and breaking changes submit pull requests to `dev`.
```
Comment (jacekkarczmarczyk) [inline packages/docs/README.md]: breaking changes should go to `next`

Comment (bad4iz) [reply packages/docs/README.md]: image

Comment (bad4iz) [reply packages/docs/README.md]: here and there then need?
image

Comment (MajesticPotatoe) [reply packages/docs/README.md]: Yeah probably should update the contribution page as well

Pull requests related to Vuetify: 
* For **bug fixes** and **documentation updates** submit pull requests to `master`.
* For **new features** and **enhancements** submit pull requests to `dev`
* For any **features** that contain **breaking changes** submit pull requests to `next`

**Comment by KaelWD** [comment]
This isn't really relevant to docs anyway, should be in the contribution guide.

**Comment by nekosaur** [comment]
I agree with kael, this information should be (and already is?) on the contribution page

**Comment by jacekkarczmarczyk** [comment]
Isn't repo/CONTRIBUTING.md better place for this?

**Comment by bad4iz** [comment]
then you need to delete it readme? 
I made the current version. It was written wrong. Same.

**Comment by MajesticPotatoe** [comment]
Probably best just to revert the readme and update the relevant info on the contribution guide page.

**Comment by bad4iz** [comment]
> * [x]  удалить README.md из `/packages/docs`
> * [x] обновить README.md `/packages`до простого списка из 4 пакетов + простое описание пакетов
> * [ ] обновить `Contributing`раздел на главной странице со ссылкой на руководство по вкладу

File: packages/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<p align="center">
+  <a href="https://vuetifyjs.com" target="_blank"><img width="100"src="https://cdn.vuetifyjs.com/images/logos/logo.svg"></a>
+</p>
+<p>This is the packages for the <a href="https://vuetifyjs.com" target="_blank">Vuetify</a>.</p>
```
Comment (MajesticPotatoe) [inline packages/README.md]: `<p>These are the packages for the <a href="https://vuetifyjs.com" target="_blank">Vuetify</a> monorepo.</p>`

File: packages/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+# Be a Part of Something Bigger!
```
Comment (MajesticPotatoe) [inline packages/README.md]: Can be removed.

**Comment by jacekkarczmarczyk** [comment]
Don't we want CONTRIBUTING.md in repo? It's much easier to maintain than docs

**Comment by KaelWD** [comment]
It already is, and it's what gets hinted at by github the first time someone opens an issue or PR https://github.com/vuetifyjs/vuetify/blob/master/.github/CONTRIBUTING.md

**Comment by jacekkarczmarczyk** [comment]
Oh, i was looking in the root folder, anyway i meant that the contents (what is in docs now and what was first added in this PR and then removed) could be in this file

**Comment by MajesticPotatoe** [comment]
@jacekkarczmarczyk we will do that as another PR. This one pretty much updated the `README.md` We got one in the works for release notes and migrations (using fetch).
we can then update the `CONTRIBUTION.md` and fetch that into the docs.

**Pull request merged**